extensions/extensions-security/extensions-policy-decision-point-service/pom.xml

```diff
@@ -85,6 +81,46 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-source-plugin</artifactId>
             </plugin>
+            <plugin>
```

Comment:
I: the plug in step of prepare package should be at last, because it was trying to archive before quarkus-app were generated.